src/LookupTable/contexts/LookUpTableImportCsvContext.tsx

```diff
@@ -49,8 +48,8 @@ const LookUpTableImportCsvReducer = (
     case LookUpTableImportCsvActions.ImportCSV:
       return { ...state, file: action.payload }
     case LookUpTableImportCsvActions.SetTableName: {
-      const removeNumbers = action.payload.replace(/([0-9])/gm, '')
-      return { ...state, tableName: camelCase(removeNumbers) }
+      const restrictCharacters = action.payload.replace(/[^A-z_]/gm, '')
```

Comment:
Name can now be any alphabetical character or `_`. So you can write names in camelCase or snake_case as you see fit -- the database table name will always be converted to snake_case, but the "label" will be exactly what you enter.